pkg/common/config.default.yaml

```diff
@@ -114,6 +114,7 @@ worker:
     #     minFreeGpu:
     #     minFreeMemory:
   # global pool attributes
+  exposeLocalNetwork: true
```

Comment:
I would rename this to something like `useHostResolvConf`. It really just allows you to use the host's DNS setup.

Reply:
updated